src/sql/workbench/contrib/notebook/browser/notebookViews/notebookviews.css

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.grid-stack {
+	margin: 0 8px;
+}
+.grid-stack>.grid-stack-item {
+	margin: 0;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content {
+	display: flex;
+	cursor: move;
+	box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.13);
+	border-radius: 2px;
+	overflow: visible;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content > .grid-stack-content-wrapper {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	margin-bottom: 20px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-wrapper-inner {
+	overflow-y: auto;
+	background: #fff;
+	height: 100%;
+	cursor: auto;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content ::-webkit-scrollbar {
+	width: 8px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-header {
+	flex-direction: row;
+	justify-content: space-between;
+	display: flex;
+	margin-bottom: 3px;
+	padding: 0 5px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-header .action-label.codicon{
+	background-size: 16px;
+	padding: 11px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-item-title {
+	font-size: 13px;
+	line-height: 25px;
+	font-weight: 400;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .actionbar {
+	top: -14px;
+	z-index: 999;
+}
+.grid-stack>.grid-stack-item.notebook-cell.active .actionbar {
+	background-color: #ffffff;
+	z-index: 1;
+	transform: translate(0px, 7px);
+}
+.grid-stack>.grid-stack-item.ui-draggable-dragging>.grid-stack-item-content,.grid-stack>.grid-stack-item.ui-resizable-resizing>.grid-stack-item-content{
+	box-shadow:1px 4px 6px rgba(0,0,0,.2);
+	opacity:.8
+}
+.grid-stack>.grid-stack-item>.ui-resizable-se,.grid-stack>.grid-stack-item>.ui-resizable-sw{
+	background-image:url("data:image/svg+xml;charset=UTF-8,%3csvg width='19' height='19' viewBox='0 0 19 19' fill='none' xmlns='http://www.w3.org/2000/svg'%3e%3cline x1='0.646447' y1='14.6464' x2='14.6464' y2='0.646447' stroke='%230078D4'/%3e%3cline x1='8.07125' y1='15' x2='15.0713' y2='7.99996' stroke='%230078D4'/%3e%3c/svg%3e");
```

Comment:
Please move the SVG into a media folder. Look at similar components and where they store their SVG.